apps/pdc-frontend/src/app/[locale]/(rootLayout)/page.tsx

```diff
@@ -112,24 +113,26 @@ const Home = async ({ params: { locale } }: { params: any }) => {
         }}
         Link={Link}
       />
-      <main id="main">
-        <Heading level={1}>{t('h1')}</Heading>
-        <Grid>
-          <>
-            <GridCell md={10} lg={9}>
-              <section>
-                <TopTask data={topTasksData?.link as TopTaskDataTypes[]} />
-              </section>
-            </GridCell>
-            <GridCell md={10} lg={9}>
-              <section>
-                <Heading2>{t('components.alphabetically-products-navigation')}</Heading2>
-                <IndexCharNav characters={alphabetAvailability} component="link" Link={IndexCharNavLink} />
-              </section>
-            </GridCell>
-          </>
-        </Grid>
-      </main>
+      <RichText>
+        <main id="main">
+          <Heading level={1}>{t('h1')}</Heading>
+          <Grid>
+            <>
+              <GridCell md={10} lg={9}>
+                <section>
+                  <TopTask data={topTasksData?.link as TopTaskDataTypes[]} />
+                </section>
+              </GridCell>
+              <GridCell md={10} lg={9}>
+                <section>
+                  <Heading2>{t('components.alphabetically-products-navigation')}</Heading2>
+                  <IndexCharNav characters={alphabetAvailability} component="link" Link={IndexCharNavLink} />
+                </section>
+              </GridCell>
+            </>
+          </Grid>
+        </main>
+      </RichText>
```

Comment:
Ik zou het omdraaien: `<main><RichText>...</RichText></main>`. Op die manier kun je ook nog iets anders dan rich text in de main content hebben, nu is alles verplicht rich text met de bijbehorende margins.

Dit geldt ook voor de andere files.